index.html

```diff
@@ -549,6 +549,14 @@
 that the person can easily switch it off. In general, providing [=consent=] should be rare,
 difficult, highly intentional, and temporary.
 
+In some situations, a [=party=] can comply with a requirement to get a [=person=] to perform
+some action that is treated as <q>consent</q> without obtaining the person's actual, informed [=consent=].
```

Comment:
I don't know of any laws that require companies to request actions that do not demonstrate consent. That many companies have skirted some laws, or interpreted laws in ways that are harmful to users, is not the same as a legal requirement to harass users with meaningless dialogs.